shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -566,7 +566,9 @@
       // All the children of this element are absolutely positioned and then
       // transformed, so manually set a height for the convenience of
       // embedders.
-      this.element.style.height = this.layout.offsetY + 'px';
+      var padding =
```

Comment:
Hmm, seems a bit fragile. Could we possibly get this same value from something like: ```this.element.getBoundingClientRect().bottom``` ? (Or even the parent of the gaia-grid element if not?